scripts/generate-certs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Define variables
+CERT_DIR="./certs"
+PRIVATE_KEY="$CERT_DIR/private.key"
+CERTIFICATE="$CERT_DIR/certificate.crt"
+DAYS_VALID=365
+
+# Create directory for certificates if it doesn't exist
+mkdir -p "$CERT_DIR"
+
+# Generate a private key and self-signed certificate
+openssl req -x509 -newkey rsa:4096 -sha256 -days "$DAYS_VALID" -nodes \
```

Comment:
dunno if we need to install `openssl` for e2e-tests or if it's in the base image already

Reply:
i expect it exists, if no we will need a separate task to include it or pr-generate certs ?